projects/plugins/jetpack/modules/widgets/top-posts.php

```diff
@@ -246,15 +302,20 @@ function widget( $args, $instance ) {
 		$types = isset( $instance['types'] ) ? (array) $instance['types'] : array( 'post', 'page' );
 
 		// 'likes' are not available in Jetpack
-		$ordering = isset( $instance['ordering'] ) && 'likes' == $instance['ordering'] ? 'likes' : 'views';
-
-		if ( isset( $instance['display'] ) && in_array( $instance['display'], array( 'grid', 'list', 'text' ) ) ) {
+		$ordering = isset( $instance['ordering'] ) && 'likes' === $instance['ordering'] ?
+			'likes'
+			: 'views';
```

Comment:
I like better how you did this just above.
```suggestion
		$ordering = isset( $instance['ordering'] ) && 'likes' === $instance['ordering']
			? 'likes'
			: 'views';
```

Reply:
Yep, that was an oversight from me. Committing.